### PR TITLE
psi-plus: Add “unclechu” to the list of maintainers

### PR DESCRIPTION
**Pull request by unclechu**
###### Motivation for this change

See https://github.com/NixOS/nixpkgs/pull/129710#issuecomment-877328671

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by unclechu** [comment]
Invite @misuzu to review

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -10868,6 +10868,16 @@
     githubId = 1607770;
     name = "Ulrik Strid";
   };
+  unclechu = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:10871]: Please add yourself as a maintainer by adding an alphabetically sorted entry to `maintainers/maintainer-list.nix` and commit this change into a separate commit with the message ``maintainers: add NAME`` into this PR.

Comment (unclechu) [reply maintainers/maintainer-list.nix:10871]: @SuperSandro2000 to clarify, it is alphabetically sorted on the top-level. It goes after `ulrikstrid`.

To make sure I don’t mess up I put my entry after last entry that start with `u`, selected all entries that are starting with `u` and applied this command in vim:

``` viml
'<,'>!perl -E '@o=(); while (<>) { if (/^  };/) { push @o, $x.$_; $x=""; } else { $x.=$_ } }; print join "", sort @o' 2>&1
```

Comment (unclechu) [reply maintainers/maintainer-list.nix:10871]: @SuperSandro2000 I force-pushed a change where I split the add to the maintainers list into a separate commit.

Comment (SuperSandro2000) [reply maintainers/maintainer-list.nix:10871]: That is a standard reply text I select when there is something about the maintainer entry not being in an extra commit or not being sorted. It was actually sorted in your case :)

**Pull request merged**